composer/algorithms/gradient_clipping/gradient_clipping.py

```diff
@@ -42,6 +42,11 @@ def apply_gradient_clipping(model: Union[ComposerModel, torch.nn.Module], clippi
         if version.parse(torch.__version__) < version.parse('1.13.0'):
             raise RuntimeError('To use FSDP with Composer, you must use torch>=1.13.0.')
         from torch.distributed.fsdp import FullyShardedDataParallel
+
```

Comment:
I suspect this pattern is used at multiple places already and will be used at more places. Should we create a function for this that caches this result?